pom.xml

```diff
@@ -250,6 +251,7 @@
                               <exclude>**/src/site/resources/**/*.txt</exclude>
                               <exclude>**/src/site/resources/**/*.json</exclude>
                               <exclude>**/.gitignore</exclude>
+                              <exclude>**/*.md</exclude>
```

Comment:
all (or nearly all) of the md files in the project have an apache license as a footer.  did you come across any that didn't?

also, we should probably consolidate these rules in the master pom, so they get applied in project and examples.